Swiftfin tvOS/Views/SettingsView/CustomizeViewsSettings/Components/Sections/HomeSection.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+
+extension CustomizeViewsSettings {
+    struct HomeSection: View {
+
+        @Default(.Customization.Home.showRecentlyAdded)
+        private var showRecentlyAdded
+        @Default(.Customization.Home.maxNextUp)
+        private var maxNextUp
+        @Default(.Customization.Home.resumeNextUp)
+        private var resumeNextUp
+
+        @State
+        private var isPresentingNextUpDays = false
+        @State
+        private var maxNextUpDays: Int
+
+        init() {
+            self._maxNextUpDays = .init(initialValue: Int(_maxNextUp.wrappedValue / 86400))
+        }
+
+        var body: some View {
+            Section(L10n.home) {
+
+                Toggle(L10n.showRecentlyAdded, isOn: $showRecentlyAdded)
+
+                Toggle(L10n.nextUpRewatch, isOn: $resumeNextUp)
+
+                ChevronButton(
+                    L10n.nextUpDays,
+                    subtitle: {
+                        if maxNextUp > 0 {
+                            return Text(
+                                Date.now.addingTimeInterval(-maxNextUp) ..< Date.now,
+                                format: .components(style: .narrow, fields: [.year, .month, .week, .day])
+                            )
+                        } else {
+                            return Text(L10n.disabled)
+                        }
+                    }()
+                )
+                .onSelect {
+                    isPresentingNextUpDays = true
+                }
+                .alert(L10n.nextUpDays, isPresented: $isPresentingNextUpDays) {
```

Comment:
To at least help with the tvOS button oddity, we can add a `Button(L10n.done) { }` button to at least help with navigation. Match in iOS as well.

Reply:
I added this at one point but ended up with 2 buttons. One with the number and one with Done.

IMO, the number button **appears** a little more intentional than a number next to a done. But also, I do NOT have a knack for UX. Let me know if you'd prefer 1 number button or a number next to a done button and I'll take that route!

Reply:
I'm a bit perplexed on why that button shows up but I also haven't tested on an actual device. There have been differences and quirks between the tvOS simulator and device in particular. Sure, let's just not add the `Done` button and maybe in the future when this issue is fixed we will get the system `OK` button.

Reply:
Both HomeSections should mirror each other now. I tacked on a TODO to look at this later. I know the next tvOS build is a ways away but I'll double check this when there is the next Testflight.